apps/src/templates/sectionsRefresh/CurriculumQuickAssign.jsx

```diff
@@ -20,6 +20,8 @@ export default function CurriculumQuickAssign({updateSection, sectionCourse}) {
   const [decideLater, setDecideLater] = useState(false);
   const [marketingAudience, setMarketingAudience] = useState(null);
 
+  const showPlOfferings = queryParams('participantType') === 'teacher';
```

Comment:
This can also equal `facilitator`, so I think this should be `queryParams('participantType') !== 'student'`

Reply:
ahh good call. Fixed!